### ISSUE TITLE
Time allows 60 mins.

### ISSUE THREAD

**Issue by piotzkhider** [spatie/opening-hours#98]
```php
Time::fromString('aa:bb'); // throw

Time::fromString('24:10'); // not thow
Time::fromString('16:60');
```

Are you both allowed on purpose?

**Comment by kylekatarnls** [comment]
Indeed, we could be more precise, minutes should never exceed 59 and maximum whole hour:minute is 24:00 (meaning 00:00 next day).

**Comment by kylekatarnls** [comment]
Fixed in [2.1.2](https://github.com/spatie/opening-hours/releases/tag/2.1.2)
